pom.xml

```diff
@@ -50,7 +50,7 @@
         <dep.airlift.version>0.148</dep.airlift.version>
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.29</dep.slice.version>
-        <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
+        <dep.aws-sdk.version>1.11.165</dep.aws-sdk.version>
```

Comment:
Having a proper s3 integration testing needs some work. We need an aws account and plenty of test code to wire things up. In the past I had access to an s3 deployment and I was able to test version upgrades like this with my custom test suite, but now I don't have such access.

And yes there is a need to test new aws sdk releases, because sometimes there are side effects of version upgrades that may impact the s3 users.